### ISSUE TITLE
Training Code for VLM Reward Model

### ISSUE THREAD

**Issue by jacklishufan** [ZiyuGuo99/Image-Generation-CoT#7]
Hi 
Thanks for the great works. Do you plan to release training code for the Llava reward model and the data (or data generation pipeline)?
Best

**Comment by LTH9898** [comment]
I’m facing the same issue and would appreciate any insights or updates.

**Comment by henbucuoshanghai** [comment]
maybe never?

**Comment by ZiyuGuo99** [comment]
Thank you for your interest in our work. We have now released the ORM and PARM. As we are currently occupied with some school-related matters, we are progressively open-sourcing the training code and data. 

We appreciate your patience and understanding.

**Comment by jacklishufan** [comment]
> I’m facing the same issue and would appreciate any insights or updates.

For folk's still looking for training data, I recommend checking our latest work [Reflect-DiT: Inference-Time Scaling for Text-to-Image Diffusion Transformers via In-Context Reflection](https://github.com/jacklishufan/Reflect-DiT) for now.
We have released the training data of a VLM reward model, which is curated following a similar pipeline to SANA-1.5 (which is also similar to Image-Generation-CoT, as they are both based on GenEval style evaluation using detectors)
